src/metrics/logic.py

```diff
@@ -237,20 +237,7 @@ def store_article_access(request, article, access_type, galley_type='view'):
                 galley_type=galley_type,
             ).exists()
 
-            if exists:
-                # get the most recent access attempt and reset its accessed to now.
-                access = models.ArticleAccess.objects.filter(
-                    identifier=identifier,
```

Comment:
I can't ping the above code but counter recommends IP + user agent as the identifier when a tracking session/ cookie is not available